airflow/templates/airflow-scheduler.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.airflow.prefix -}}scheduler
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: scheduler
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: {{ .Values.airflow.prefix -}}scheduler
+        image: {{ .Values.airflow.image }}
+        volumeMounts:
```

Comment:
I think you need to conditionalize this with `.Values.airflow.persistence.enabled`